settings.gradle.kts

```diff
@@ -0,0 +1,18 @@
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+pluginManagement {
+    repositories {
+        google()
+        gradlePluginPortal()
+        mavenCentral()
+    }
+}
+
+dependencyResolutionManagement {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+rootProject.name = "unfurl-kmp"
```

Comment:
Let's keep it `unfurl`?

Reply:
My bad. AS does not allow you to keep the same parent and submodule name while creating a project.
hence had to change it